helmfile/charts/prometheus-alerts/templates/rules/predict-linear.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.predictLinear}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-alerts.fullname" .) "predict-linear" | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ template "prometheus-alerts.name" . }}
+{{ include "prometheus-alerts.labels" . | indent 4 }}
+{{- if .Values.defaultRules.labels }}
+{{ toYaml .Values.defaultRules.labels | indent 4 }}
+{{- end }}
+{{- if .Values.defaultRules.annotations }}
+  annotations:
+{{ toYaml .Values.defaultRules.annotations | indent 4 }}
+{{- end }}
+spec:
+  groups:
+  - name: predict-linear
+    rules:
+    - alert: PersistentVolumeSixtySixProcentInThreeDays
+      annotations:
+        message: The PersistentVolume claimed by {{`{{ $labels.persistentvolumeclaim }}`}} in Namespace {{`{{ $labels.namespace }}`}} is full in three days.
+      expr: predict_linear(kubelet_volume_stats_available_bytes{job="kubelet"}[24h], 3*24*60*60) <= (kubelet_volume_stats_capacity_bytes*(1-{{ .Values.predictLinearLimit }}))
```

Comment:
For my understanding, why `job="kubelet"`?

Reply:
Removed it now as I realize it's not needed and I also didn't use that argument in the command after which should cause issues.